### PR TITLE
Animal animation v2

### PR DESCRIPTION
**Pull request by AlexanderSelmanovic**
Walk and Idle animations added to all Animals, new prefabs added

### CODE REVIEW

File: Assets/Scripts/Animation/EntityAnimation.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using Ecosystem.ECS.Hybrid;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace Ecosystem.Animation {
+    public class EntityAnimation : MonoBehaviour {
```
Comment (gimme) [inline Assets/Scripts/Animation/EntityAnimation.cs:7]:
```suggestion
    [RequireComponent(typeof(Animator))]
    public class EntityAnimation : MonoBehaviour {
```

File: Assets/Scripts/Animation/EntityAnimation.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        private Animator animator;
+
+        [SerializeField]
+        private Movement movement = default;
+
+        void Start() {
```
Comment (G1mmicK) [inline Assets/Scripts/Animation/EntityAnimation.cs:12]:
```suggestion
        void Awake() {
```

File: Assets/Scripts/Animation/EntityAnimation.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                RunAnimation();
+            } else {
+                IdleAnimation();
+            }
+        }
+
+        private void IdleAnimation() {
+            if (animator != null) {
+                animator.SetBool("Run", false);
+            }
```
Comment (gimme) [inline Assets/Scripts/Animation/EntityAnimation.cs]:
```suggestion
            animator.SetBool("Run", false);
```
RequireComponent löser det här

File: Assets/Scripts/Animation/EntityAnimation.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        }
+
+        private void RunAnimation() {
+            if (animator != null) {
+                animator.SetBool("Run", true);
+            }
```
Comment (gimme) [inline Assets/Scripts/Animation/EntityAnimation.cs]:
```suggestion
            animator.SetBool("Run", true);
```

**Review by gimme — CHANGES_REQUESTED** [review]

**Review by gimme — APPROVED** [review]

**Pull request merged**